pkg/skaffold/runner/runcontext/context.go

```diff
@@ -235,8 +235,9 @@ func GetRunContext(ctx context.Context, opts config.SkaffoldOptions, configs []s
 		return nil, fmt.Errorf("getting current cluster context: %w", err)
 	}
 	kubeContext := kubeConfig.CurrentContext
-	log.Entry(context.TODO()).Infof("Using kubectl context: %s", kubeContext)
 
+	fmt.Printf("Using kubectl context: %s\n", kubeContext)
+	fmt.Printf("Using cluster: %s\n", kubeConfig.Contexts[kubeContext].Cluster)
```

Comment:
So we should use `output.Default` to get the default colouring of console messages:
```
	output.Default.Fprintf(out, "Using kubectl context: %s\n", kubeContext)
	output.Default.Fprintf(out, "Using cluster: %s\n", kubeConfig.Contexts[kubeContext].Cluster)
```

That'll require updating callers to pass through the output `io.Writer`.

I then tried running `skaffold dev --kube-context blahblah` and got a panic as I had no `blahblah` context!  We apparently were never validating the contexts 😱   Would you be up for modifying the `CurrentConfig()` to validate the context and returning an error if not found?

Reply:
I was expecting this to happen :) I'll give it a go and let you know if I get stuck. 

Are both changes OK within the same commit?